docs/QUICKSTART.md

```diff
@@ -41,9 +41,10 @@ to the client in the `tufclient` directory.
 
 We'll create a target file that will later be delivered as an update to clients.
 Metadata about that file will be created and signed, and added to the
-repository's metadata.
+repository's metadata. 
```

Comment:
@lukpueh done.
I tried to test TUF in my project. I encountered this problem(I changed to data directory and I thought TUF shall remember the repo folder). It may be a trap for beginners like me:). So in my opinion it's necessary. You are expert of it, but not everyone is expert. People may think it's hard to use or it has bugs..